DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs

```diff
@@ -112,13 +116,25 @@ public class AccountController : BaseApiController
                 return BadRequest(ModelState);
             }
 
+            ActivityType activityType = _activityWriterService.GetActivityTypeByName(ActivityTypeNames.AddRole);
```

Comment:
I see a lot of code duplication. If this general to all action methods may be it can be extracted as cross cutting functionality?